auth0_flutter/pubspec.yaml

```diff
@@ -1,16 +1,14 @@
 name: auth0_flutter
-description: A new flutter plugin project.
-version: 1.0.0-beta.0
+description: Auth0 SDK for Flutter
+version: 1.0.0-beta.1 
 homepage: https://github.com/auth0/auth0-flutter
-publish_to: none
 
 environment:
   sdk: ">=2.17.0 <3.0.0"
   flutter: ">=3.0.0"
 
 dependencies:
-  auth0_flutter_platform_interface:
-    path: ../auth0_flutter_platform_interface
+  auth0_flutter_platform_interface: 1.0.0-beta.1
```

Comment:
Packages without `publish_to` fields cannot have dependencies using `path`, so we will have to depend on the published version.

Reply:
This may mean that the build will fail the first time before we publish `auth0_flutter_platform_interface`.

Reply:
Actually, I've reverted that change in d7e5e99 and will raise another release PR for the `auth0_flutter` package.